soa_proposal.md

```diff
@@ -0,0 +1,30 @@
+# Proposal
+
+- We rename the event
+- We keep the event completely open.
+- Each participant is in one of three phases at any given time.
+- Every time a participant finishes a phase, they are moved to the next phase.
+
+## Phase 1 participants
+
+- We set up a simple to follow tutorial for python/any other language we can support.
+- This needs to be a 10 page setup MAX
+- The idea is to be as lightweight as possible so that people want to keep coming back to the event and tell their friends about it.
+- Use repl.it to set up the teaching IDE.
+- Do basic concepts like math, lists, dict, loops, print/input
+- Each concept / section has 2 questions as test. Once all tests are done the participant has cleared the phase.
+- We can now give them links to python docs, practice sites etc. to read in their free time.
+
+##  Phase 2 participants
+
+- We shift them from repl.it to hackerrank/leetcode/hackerearth easy questions
+- We provide them a few questions as practice along with tutorials for that.
+- After that we give them a target. For example 5 start python on hackerrank or something else...
+- In order to achieve that target participants are encouraged to ask questions in a group setting.
+- We use github issues to manage these discussions. This preps them to use github for the next phase.
+
+## Phase 3 participants
```

Comment:
Phase 3 could be harder to maintain.

Reply:
yeah. I was thinking that only 2-5 people are going to reach phase 3 so it should be ok.